### PR TITLE
Python API for robocorp.com/docs

### PR DESCRIPTION
**Pull request by osrjv**
First simple version of building markdown for Python APIs and ingesting into robocorp.com/docs.

Changes:
- Adds a new `docs.python-markdown` invoke target
- Builds a `python.json` file into rpaframework.org root
- Change `python.rst` files to only target main class (more simple to ingest _at this point_)
- Remove docs-related dev dependencies from individual packages
- Minor formatting fixes in various libraries

### CODE REVIEW

File: pyproject.toml
Comment (mikahanninen) [inline pyproject.toml:35]: Any specific reason why not to use the latest 0.16.0 ?

Comment (osrjv) [reply pyproject.toml:35]: I think might have been a merge/rebase oversight! I'll bump it

**Comment by cmin764** [comment]
Might be cool to merge this after merging & releasing #1110. (will do a review soon)

File: .github/workflows/dialogs.yaml
Comment (cmin764) [inline .github/workflows/dialogs.yaml:1]: Maybe this were supposed to remove Python3.7 not 3.8, now that I refactored these into supporting 3.8 -> 3.10 instead of the old 3.7 -> 3.9.

**Review by cmin764 — APPROVED** [review]
Apart from a small notice in our deprecated `dialogs` package, LGTM!

**Pull request merged**
